src/com/amazon/ionelement/impl/LobElementBase.kt

```diff
@@ -36,9 +36,9 @@ internal abstract class LobElementBase(
     override fun equals(other: Any?): Boolean {
         return when {
             this === other -> true
-            other !is LobElementBase -> false
+            other !is LobElement -> false
             type != other.type -> false
-            !bytes.contentEquals(other.bytes) -> false
+            bytesValue != other.bytesValue -> false
```

Comment:
🗺️ This is now comparing the `ByteArrayView` rather than comparing the `ByteArray` directly (since we can no longer be sure that `other` is backed by a `ByteArray`).